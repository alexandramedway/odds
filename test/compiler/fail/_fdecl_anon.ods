do get_add = () -> return (x, y) -> return x + y
do sum = get_add()
do sum(40, true)
