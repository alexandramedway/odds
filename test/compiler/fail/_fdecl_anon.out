Analyzer Exception: Function 'sum' expected argument of type Num but was passed Bool instead
